indico/htdocs/sass/modules/_abstracts.scss

```diff
@@ -16,3 +16,7 @@
  */
 
 @import 'abstracts/abstract_page';
+
+.submitter-column {
```

Comment:
Agreed, I'd also rather add a new file in `abstracts/` instead of mixing both in `_abstract.scss`